src/parser/mod.rs

```diff
@@ -2231,6 +2231,18 @@ impl<'a> Parser<'a> {
         ))
     }
 
+    fn parse_duckdb_struct_type_def(&mut self) -> Result<Vec<StructField>, ParserError> {
+        self.expect_keyword(Keyword::STRUCT)?;
+        self.expect_token(&Token::LParen)?;
+        let field_defs = self
+            .parse_comma_separated(Self::parse_struct_field_def)?
+            .into_iter()
+            .map(|(f, _)| f)
```

Comment:
Oh this looks like a bug for us to silently drop the returned flag since there might be an extraneous `>`, seems like we would instead do something like this
```rust
let field_defs = self.parse_comma_separated(|self| {
    Ok(StructField {
        field_name: Some(self.parse_identifier(false)?), // I assume field name isn't optional for duckdb?
        field_type: self.parse_data_type(),
    })